### DISCUSSION TITLE
504 Error

### DISCUSSION THREAD

**Discussion by redeio** [spencerwooo/onedrive-vercel-index#247] [Q&A] [answered]
Request failed with status code 504

**Comment by spencerwooo** [comment, marked as answer]
the code present on your fork is not the latest version, you should fork this repo instead

**Reply by redeio** [reply]
It's the one that forked your homepage.

**Reply by spencerwooo** [reply]
i dont think you have read the deployment docs here: https://github.com/spencerwooo/onedrive-vercel-index#new-users

you should not change the client id and secret in api.json, that is the whole purpose of the new version - not having to generate your own client id/secret, you should only:

- change site.json - specifically the baseDirectory and the userPrincipalName
- use upstash to get a redis db
- add the redis url to your env var
